### ISSUE TITLE
Problematic Build Configuration Allocated Memory textbox

### ISSUE THREAD

**Issue by popcorn9499** [pterodactyl/panel#4303]
### Current Behavior

if i type say "1,024" to the Allocated Memory box in Resource Management box that "1,024" magically becomes "1"

### Expected Behavior

If i type say "1,024" to the Allocated Memory box it does one of two things. either the coma getting stripped and it being "1024" orr an error message saying something like this "The memory must be a number."

### Steps to Reproduce

go to the Build Configuration tab on any game server then change the Allocated Memory box to say "1,024" and then hitting "Update Build Configuration"  you should now see "1" in the Allocated Memory box.

### Panel Version

1.10.1

### Wings Version

1.7.0

### Games and/or Eggs Affected

Minecraft(Paper)

### Error Logs

```bash
wings: https://ptero.co/ipagekelil
panel: returned ""
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by QuintenQVD0** [comment]
1024 mb = 1 Gb it is automatic converted.

**Comment by QuintenQVD0** [comment]
Also in mb you should never use a comma. 
1 Gb = 1024 mb
2 Gb = 2048 mb 
3 Gb = 4072 mb

**Comment by popcorn9499** [comment]
I would agree you should never use a comma but I accidentally did and noticed the inconsistency between the new server creation wizard and the build configuration menu, since I get notified of the error in the new server creation wizard.

It shouldn't silently hide an error like this in this way

**Comment by Boy132** [comment]
There is no error "hidden" here.

The `,` is seen as the decimal separator. (same for `.`)
So `1,024` is _not_ "one thousand and twenty four" but "one with decimals zero two four" - so rounded just "one".

**Comment by popcorn9499** [comment]
what happens under "Build Configuration" [video](https://photos.popicraft.net/u/files/10.17.26-06.08.22-chrome_J0AgVS5uAb.mp4)

what happens under the server creation wizard [video](https://photos.popicraft.net/u/files/10.18.49-06.08.22-chrome_PLL8CyBxcA.mp4) 

This is why it felt like more of a bug since the behavior here is inconsistent
